test/metric_value_benchmark/agent_configs/disk_config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "agent": {
+    "metrics_collection_interval": 60,
+    "run_as_user": "root",
+    "debug": true,
+    "logfile": ""
+  },
+  "metrics": {
+    "namespace": "MetricValueBenchmarkTest",
+    "append_dimensions": {
+      "InstanceId": "${aws:InstanceId}"
+    },
+    "metrics_collected": {
+      "disk": {
+        "resources": [
+          "*"
+        ],
+        "measurement": [
+          "free",
+          "inodes_free",
+          "inodes_total",
+          "inodes_used",
+          "total",
+          "used",
+          "used_percent"
```

Comment:
Instead of just testing a single JSON config with ALL metrics enabled, it would be better to iteratively test combinations of metrics.
1. only configure metric A, and verify only A is reported with recent timestamp.
2. only configure metric B, and verify only B is reported with recent timestamp.
3. .... repeat for metrics A-Z.
4. Then configure metric A and B, and verify only those 2 get reported.
5. I realize the number of combinations will be too large and make test time a pain, so please think of ways to reduce the combinations... Maybe iterate from 0 to total_num_metrics, and than randomly select current_count number of metrics.

Reply:
@adam-mateen that's an interesting point. I'll convert that into a github issue so that we can contain our discussion over there

Reply:
Issue #71

Reply:
Interesting point. However, why all combinations though? All combinations are the subset of testing all metrics? Unless there are higher gate to deal with that plugins (e.g testing small subset of metrics in first part and testing others metrics in performance metrics)

Reply:
please discuss in #71 since this is out of scope